config/rbac/hive_backend_role.yaml

```diff
@@ -0,0 +1,24 @@
+# hive-backend is a role intended to enable groups running hive to have
+# debug access when investigating issues (whether the issue is with
+# hive or with the cluster(s) being managed).
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: hive-backend
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
```

Comment:
Are we absolutely sure that we want to grant read access to all of the secrets in the cluster to the hive-backend user?

Reply:
One of the use cases was to have access to the kubeconfig for the managed clusters 🤷

Reply:
My initial reaction was "this will not be allowed in OSD." Then I saw the ticket was created by SRE-P for backplane.

/cc @jaybeeunix @jewzaam @wanghaoran1988 @jmelis please advise.